### PR TITLE
feat(i18n): add Romanian translations.

### PR DESCRIPTION
**Pull request by nickvasdev**
This improves localization by adding Romanian translations.

### CODE REVIEW

File: locales/ro.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "Path to user npmrc.": "Calea către fișierul npmrc al utilizatorului.",
+  "Execute string as if inside `npm run-script`.": "Execută string-ul ca și cum ar fi rulat în `npm run-script`.",
+  "Shell to execute the command with, if any.": "Shell pentru a executa o comandă, dacă există.",
+  "Generate shell code to use npx as the \"command not found\" fallback.": "Generează cod shell pentru a folosi npx drept fallback pentru \"command not found\".",
+  "Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.": "Ignoră executabilele binare din $PATH, sau din proiectul local. Aceasta forțează npx să instaleze și să folosească ultima versiune temporar.",
+  "npm binary to use for internal operations.": "Executabil binar npm pentru operaţiuni interne.",
+  "For the full documentation, see the manual page for npx(1).": " Pentru documentația completă, verifică manualul la pagina specifică pentru npx(1).",
+  "Unable to guess a binary name from %s. Please use --package.": " Imposibil de ghicit numele executabilului binar din %s. Vă rugăm să folosiți --package.",
```
Comment (zkat) [inline locales/ro.json]: Are the spaces at the beginning of these two necessary?

**Review by zkat — COMMENTED** [review]
Just a tiny question but otherwise looks good

File: locales/ro.json (hunk 1 above)
Comment (nickvasdev) [reply locales/ro.json]: @zkat They're not, I just missed them, sorry. I updated my latest commit. :)

**Comment by zkat** [comment]
Yay! Thanks!

**Pull request merged**